README.wcf.md

```diff
@@ -0,0 +1,110 @@
+# Featured Tags
```

Comment:
@HongGit I'm not sure why there are differences, could you resolve this?

Reply:
@MichaelSimons I thought that you would like the Readme to match the .NET framework style, therefore I made the change.  Can you please elaborate a bit more?

Reply:
This readme does not match the recommended MCR structure.  It appears to vary a fair amount from the .NET framework style (see https://hub.docker.com/_/microsoft-dotnet-framework-runtime/).  

1. The **Related Repos** section is very different both in content and location.  
1. There should be an **About This Image** section instead of **What is WCF**

I do like the changes to align the **Support** and **Feedback** sections.